internal/handlers/hana/msg_drop.go

```diff
@@ -16,12 +16,60 @@ package hana
 
 import (
 	"context"
+	"errors"
 
+	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
```

Comment:
Hana handler should not import PostgreSQL database package. The linter is correct there, please run it locally with `task lint`.